lang/machine.md

```diff
@@ -91,6 +95,15 @@ impl<M: Memory> Machine<M> {
             mem.store(global_ptrs[global_name], bytes, global.align)?;
         }
 
+        // Allocate functions.
+        for (fn_name, _function) in prog.functions {
+            let alloc = mem.allocate(Size::ZERO, M::FN_ALIGN)?;
```

Comment:
Why do you hard-code the size to 0 but the alignment is a parameter? Might as well hard-code both of them at that point. Also I don't think `8` is right, functions do not have any alignment requirements to my knowledge.